src/util/BaseLoginRouter.js

```diff
@@ -106,6 +107,19 @@ function (Okta, Backbone, BrowserFeatures, RefreshAuthStateController, Settings,
         settings: this.settings
       });
 
+      if (this.settings.get('registration.click')) {
+        var registrationContainer = new RegistrationSection({
+          settings: this.settings,
+          click: this.settings.get('registration.click'),
+          label: this.settings.get('registration.label'),
+          text: this.settings.get('registration.text')
+        });
+        var wrapperId = this.el;
```

Comment:
As mentioned on Friday, we should not be doing this. For example, is the intention that this link shows up on the forgot password page, the MFA page, etc?

If we were going to do it (and I still don't think we should - talk to UX), at the very least it should be in the container or header (renaming it to some sort of container) rather than doing a raw append to the dom.